src/_locales/en/messages.json

```diff
@@ -0,0 +1,109 @@
+{
+  "launchInExternalBrowser": {
+    "message": "Open page in $browser$",
```

Comment:
Always add a comment when a string has a placeholder/variable (this applies to the entire file).

I would also keep the non-flattened structure per documentation, and add an example
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization

```
  "launchInExternalBrowser": {
    "message": "Open page in $browser$",
    "description": "The variable is replaced by the browser name (Firefox, Chrome)",
    "placeholders": { 
      "browser": { 
        "content": "$1",
        "example": "Firefox"
      } 
    }
  }
```

Reply:
PR (for descriptions): https://github.com/mozilla/firefox-launch/pull/21